zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/SearchQueryResponseMapper.java

```diff
@@ -65,4 +68,51 @@ public static Either<ProblemDetail, ProcessInstanceItem> toProcessInstance(
             .startDate(p.startDate())
             .endDate(p.endDate()));
   }
+
+  public static Either<ProblemDetail, UserTaskSearchQueryResponse> toUserTaskSearchQueryResponse(
```

Comment:
🔧 There is no case for this method to return an `Either.Left`. Thus, simply return the success type. The same goes for the mapping methods called from this method.